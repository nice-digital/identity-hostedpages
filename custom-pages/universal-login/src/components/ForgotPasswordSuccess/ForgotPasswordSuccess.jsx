import { Link } from "react-router-dom";
import { Grid, GridItem } from "@nice-digital/nds-grid";

import styles from "./ForgotPasswordSuccess.module.scss";

export const ForgotPasswordSuccess = () => (
    <div>
		<h2 className="mt--0">Thank you</h2>
		<p className="lead" data-qa-sel="forgotPasswordSuccess-message1">
			We have sent you an email with a link to help you reset your password. It might take a few minutes to come through, but if you don't receive it please check your spam folder.
		</p>
		<p className="lead" data-qa-sel="forgotPasswordSuccess-message2">If you have problems resetting your password, contact us.</p>
    	<Grid>
			<GridItem cols={12} md={9}>
				<div className={`${styles.summaryList} pv--c`}>
					<span className={styles.summaryListLabel}>
						Email
					</span>
					<span className={styles.summaryListDetail}>
						nice@nice.org.uk
					</span>
				</div>

				<div className={`${styles.summaryList} pv--c mb--d`}>
					<span className={styles.summaryListLabel}>
						Telephone
					</span>
					<span className={styles.summaryListDetail}>
						+44(0)300 323 0140
					</span>
				</div>
			</GridItem>
		</Grid>
    	<Link data-qa-sel="forgotPasswordSuccess-link-to-login" to="/">Return to sign in</Link>
    </div>
);
