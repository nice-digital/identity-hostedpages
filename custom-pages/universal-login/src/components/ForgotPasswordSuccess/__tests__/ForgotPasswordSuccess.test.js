import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";
import { ForgotPasswordSuccess } from "../ForgotPasswordSuccess";

describe("ForgotPasswordSuccess components", () => {
  it("should render ForgotPasswordSuccess correctly", () => {
		const { container } = render(
			<MemoryRouter future={{ v7_relativeSplatPath: true, v7_startTransition: true }}>
				<ForgotPasswordSuccess />
			</MemoryRouter>
		);
    expect(container).toMatchSnapshot();
  });
});
