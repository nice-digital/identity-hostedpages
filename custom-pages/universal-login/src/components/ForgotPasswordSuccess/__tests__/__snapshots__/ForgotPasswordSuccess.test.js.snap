// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForgotPasswordSuccess components should render ForgotPasswordSuccess correctly 1`] = `
<div>
  <div>
    <h2
      class="mt--0"
    >
      Thank you
    </h2>
    <p
      class="lead"
      data-qa-sel="forgotPasswordSuccess-message1"
    >
      We have sent you an email with a link to help you reset your password. It might take a few minutes to come through, but if you don't receive it please check your spam folder.
    </p>
    <p
      class="lead"
      data-qa-sel="forgotPasswordSuccess-message2"
    >
      If you have problems resetting your password, contact us.
    </p>
    <div
      class="grid"
    >
      <div
        data-component="grid-item"
        data-g="12 md:9"
      >
        <div
          class="summaryList pv--c"
        >
          <span
            class="summaryListLabel"
          >
            Email
          </span>
          <span
            class="summaryListDetail"
          >
            nice@nice.org.uk
          </span>
        </div>
        <div
          class="summaryList pv--c mb--d"
        >
          <span
            class="summaryListLabel"
          >
            Telephone
          </span>
          <span
            class="summaryListDetail"
          >
            +44(0)300 323 0140
          </span>
        </div>
      </div>
    </div>
    <a
      data-qa-sel="forgotPasswordSuccess-link-to-login"
      href="/"
    >
      Return to sign in
    </a>
  </div>
</div>
`;
